### ISSUE TITLE
swiftformat adds unnecessary space after `?` in ternary operator expression

### ISSUE THREAD

**Issue by garnett** [nicklockwood/SwiftFormat#72]
Tested on 0.20

```diff
-hasBusiness ? view?.showEditBusinessProfile() : view?.showCreateBusinessProfile()
+hasBusiness ? view? .showEditBusinessProfile() : view?.showCreateBusinessProfile()
```

**Comment by nicklockwood** [comment]
Thanks - I'll look into this.

**Comment by nicklockwood** [comment]
Fixed in 0.21
